.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Could you remove `.idea`?

Reply:
Absolutely. Also, do the paths under the librispeech and timit currently need to be changed to test_waves? Or is it better to keep test_wavs?

Reply:
> Absolutely. Also, do the paths under the librispeech and timit currently need to be changed to test_waves? Or is it better to keep test_wavs?

Please use the original folder name from the hugging face repo.